params/config_test.go

```diff
@@ -22,6 +22,81 @@ import (
 	"testing"
 )
 
+func TestIsByzantiumSanity(t *testing.T) {
```

Comment:
These need to be cleaned and improved; better error handling, more reasoning. Just stream-of-consciousness sanity checks now.